.eslintrc

```diff
@@ -0,0 +1,62 @@
+{
+  "env": {
+    "node": true
```

Comment:
This is like a template for `"globals"`, automatically populating it with the globals expected to be in Node. It's probably why the linter isn't complaining about the [use of `module.exports`](https://github.com/plaid/sanctuary/blob/master/index.js#L161).